src/coreclr/vm/i386/jitinterfacex86.cpp

```diff
@@ -442,6 +443,13 @@ void *JIT_TrialAlloc::GenBox(Flags flags)
     // jne              noAlloc
```

Comment:
You can delete the `enum_flag_Unrestored` flag check in this one as well.

Reply:
Yep. The x86 stubs require me to debug so I am deferring them until I am done with x64 on coreclr/mono.